src/collections/Integrations/docker.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Docker
+---
+
+import docker from "../../assets/images/socialIcons/docker.svg";
+import IndividualIntegrationPage from "../../sections/Meshery/Meshery-integrations/Individual-Integration/index.js"
+
+<IndividualIntegrationPage data={
+  {
+    name:"Docker",
+    img:docker,
+    text:"The easiest way to get a production-grade Kubernetes cluster up and running",
+    featureList:["Faster and easier configuration","Application isolation","Routing Mesh"],
+    docURL:"https://docs.meshery.io/installation/platforms/docker",
+    faqs:[
+    {
+      question: "How to Meshery Extension Points work?",
+      ans: "Yes, as the service management plane, Meshery, provisions and manage the lifecycle of service meshes.",
+    },
+    {
+      question: "How can I request a new integration?",
+      ans: "Yes, as the service management plane, Meshery, provisions and manage the lifecycle of service meshes.",
+    },
+    {
+      question: "Does Meshery manage service meshes?",
+      ans: "Yes, as the service management plane, Meshery, provisions and manage the lifecycle of service meshes.",
+    },
+  ]
+  }
+}/>
```

Comment:
If we would have wanted to render the data in this way itself, then why take the pain of using MDX files.
The body should only have the description of the integration and nothing more.
Rest all information should be present in the frontmatters.

Reply:
Can you give an example of that?

Reply:
Ok, the format should be this:
```
---
title:
subtitle:
heroImg:
features: [..., ...., ...] (Can be written on each line)
---

<Import the button component />
Write the body here
```

Reply:
Okay!